### ISSUE TITLE
Fix 'Empty Button' errors ('User Officer' role)

### ISSUE THREAD

**Issue by JEThompson97** [UserOfficeProject/stfc-user-office-project#115]
There is [already a separate issue](https://github.com/UserOfficeProject/stfc-user-office-project/issues/111) for fixing Empty Button errors for 'User' areas of the app.
See that issue for more context on this type of error.

This issue is to extend these fixes to the 'User Officer' areas of the app.

(These fixes were split between User and User Officer issues for manageability)

**Comment by TickleThePanda** [comment]
Blocked by #116

**Comment by TickleThePanda** [comment]
These errors might not exist any more because of the changes for #191
